### PR TITLE
PR #15 Level Design (level 7)

### PR DESCRIPTION
**Pull request by sfazli96**

### CODE REVIEW

**Review by sentrychris — CHANGES_REQUESTED** [review]
Thank you for your contribution!

This is a great design, great work! I love it.

If I could suggest just 2 small changes:

- The walls should not create blocked exits or "closed rooms", unfortunately the player will be stuck in the top left corner with the current design, as there is no way out of the room. Maybe change the top left corner to look like this instead:
image

- There should be fewer pickup items (Ammo (A), Health (H), Stamina (S)) to maintain the overall difficulty, I would recommend the following number of items based on the previous levels:
    - Ammo: 14
    - Health: 8
    - Stamina: 8

Thank you again!

File: src/browser/lib/Levels/level7.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * 14 Ammo
+ * 07 Health
+ * 07 Stamina
+ */
+export const Level7 = [
+  'W W W W W W W W W W W W W W W W W W W W W W W W W W',
+  'W               W                 W   W   W   W   W',
+  'W   A A   P     W   E   W   E     W W W   H   A   W',
+  'W               W   A   W   A   A   W   A   W   A W',
+  'W W W W W W W W W   S   W   S   S W W W   A   S   W',
```
Comment (sentrychris) [inline src/browser/lib/Levels/level7.js:14]: Maybe create an opening here e.g.

image

Comment (sfazli96) [reply src/browser/lib/Levels/level7.js]: Ok sounds good I can fix those changes and do a new pull request.

**Comment by sentrychris** [comment]
That's perfect! Thank you for contributing level 7! I'll merge it in and make sure you're credited for it in the next v0.5.1 release.

Have a great day! You're more than welcome to contribute again in the future if you'd like.

**Pull request merged**